test/server.go

```diff
@@ -40,7 +40,7 @@ func CoreDNSServerPorts(i *caddy.Instance, k int) (udp, tcp string) {
 	if t != nil {
 		tcp = t.String()
 	}
-	return
```

Comment:
Why this change?

Reply:
it must return udp and tcp right?

Reply:
Also this test case is running in my local but failing on github, just checking everything where a memory error or nil pointer error can occur.

Reply:
go doesn’t require listing the return variables if they are named in the signature. Also, it’s unrelated to plugin order, so doesn’t belong in this PR.

Reply:
okk will remove this now.

Reply:
this test is running successfully on my local and failing on github any specific reason, if you know else i will figure it out.